doc/installation.rst

```diff
@@ -73,15 +73,15 @@ the use of RedisAI_. The following libraries are supported.
      - Python Versions
      - Built By Default
    * - PyTorch_
-     - 1.7
+     - 1.11.0
      - 3.7 - 3.9
      - Yes
    * - Tensorflow_ / Keras_
-     - 2.5.2
+     - 2.8.0
      - 3.7 - 3.9
      - Yes
-   * - ONNX_
-     - 1.9
+   * - ONNX_ Runtime
+     - 1.11.1
```

Comment:
If these updates require RAI v1.2.7, we shouldn't claim that we still support older versions. There is some ambiguity here as to whether we're pushing everything forward or not.

Reply:
run_tests.yml still tries to test with older versions of RedisAI

Reply:
For now, we are keeping 1.2.3, 1.2.5, 1.2.7. I've updated the table to reflect the various versions we support